source/leuven/services/openondemand.rst

```diff
@@ -191,136 +226,244 @@ of using any of the other partitions. There are however some limitations on the
+to avoid clogging your ``$VSC_HOME``.
+You can do this by using the ``lib`` argument for both the ``install.packages`` and the ``library`` function.
 
 **Remarks:**
 
-- Navigating between your different directories is possible using the file explorer. If you are navigating by clicking the folder, you will notice that you can see all user folders. You do not have access to these, and you will receive an error when you try to open them. You will also notice that you cannot use the same way of navigating after this. Another solution is to click the three dots on the right (...) and enter your path.
-- The 'Tools-Install packages' interface does not allow you to select any other path than the default in your ``$VSC_HOME``. It is recommended to use the ``install.packages`` function instead.
-- RStudioServer will by default store the RStudio cache in ``$VSC_HOME/.local/share/rstudio``. This cache can get very large and cause you to  exceed the quota of your home directory. To avoid this, you can redirect this cache to your data directory by setting ``$XDG_DATA_HOME`` variables in your ``~/.bashrc``. 
+- Navigating between your different directories is possible using the file explorer.
+  If you are navigating by clicking the folder, you will notice that you can see all user folders.
+  You do not have access to these, and you will receive an error when you try to open them.
+  You will also notice that you cannot use the same way of navigating after this.
+  Another solution is to click the three dots on the right (...) and enter your path.
+- The 'Tools-Install packages' interface does not allow you to select any other path than the default in your ``$VSC_HOME``.
+  It is recommended to use the ``install.packages`` function instead.
+- RStudioServer will by default store the RStudio cache in ``$VSC_HOME/.local/share/rstudio``.
+  This cache can get very large, and cause you to exceed the quota of your home directory.
+  To avoid this, you can redirect this cache to your data directory by setting ``$XDG_DATA_HOME``
+  variables in your ``~/.bashrc``.
 
   .. code-block:: bash
-    
+
     echo "export XDG_DATA_HOME=$VSC_DATA/.local/share" >> ~/.bashrc
 
+
 Tensorboard
 -----------
 
-Tensorboard is an app that allows you to visualize and measure different aspects of your machine learning workflow. Have a look at the `official guidelines <https://www.tensorflow.org/tensorboard/get_started>`_ for more detailed information. 
+Tensorboard is an app that allows you to visualize and measure different aspects of your machine learning workflow.
+Have a look at the `official guidelines <https://www.tensorflow.org/tensorboard/get_started>`_ for more detailed information.
+
+The Tensorboard interactive session requires you to specify a project (or log) directory in your submission options.
+This is a relative directory starting from your ``$VSC_DATA``.
+It is not possible to navigate to the correct folder from within the app.
```

Comment:
I don't really understand this sentence (in particular what 'correct folder' means here).

Reply:
Actually, this is a what we have on the official docs, since this page was created/approved. I also do not fully get it.
But, let me ping @w-lampaert for his idea on this.

Reply:
The input data for tensorboard are always log files from a ML run. These get stored in a log directory by default. The Tensorboard app form has a project/log directory variable to specify, which should be the path to the log directory with the data you want to analyze. If you do not specify the correct directory here, the Tensorboard app will launch, but immediately give an error. Once within the session, you cannot change your working directory anymore. You can only cancel the job and start a new one with the correct directory specified under the project/log dir variable.